Snakefile

```diff
@@ -55,47 +64,60 @@ rule all:
         ],
 
 
-def devtools_install():
-    if config["ploidetect_local_clone"] and config["ploidetect_local_clone"] != "None":
-        install_path = config["ploidetect_local_clone"].format(**config)
-        devtools_cmd = f"devtools::install_local('{install_path}', force = TRUE)"
-    else:
-        ver = config["ploidetect_ver"]
-        devtools_cmd = f"devtools::install_github('lculibrk/Ploidetect', ref = '{ver}')"
-    return f'"{devtools_cmd}"'
-
 rule download_cytobands:
     """Downloads cytoband data for plotting & (todo) hgver-specific centromere filtering"""
     input:
         HTTP.remote(
-            expand("http://hgdownload.cse.ucsc.edu/goldenpath/{hgver}/database/cytoBand.txt.gz", hgver = config["genome_name"])
-        )
+            expand(
+                "http://hgdownload.cse.ucsc.edu/goldenpath/{hgver}/database/cytoBand.txt.gz",
+                hgver=config["genome_name"],
+            )
+        ),
     output:
-        expand("resources/{hgver}/cytobands.txt", hgver = config["genome_name"])
+        expand("resources/{hgver}/cytobands.txt", hgver=config["genome_name"]),
     resources:
         cpus=1,
         mem_mb=MEM_PER_CPU,
+    conda:
+        "conda_configs/sequence_processing.yaml"
```

Comment:
gzip should be on all linux/osx, so I don't think we need to specify a container/env here?

Reply:
Call me overly cautious, but I'd run it in a container anyways. It'll make it so the workflow is that much more portable/reproducible. It'll also save you down the road when inevitably someone tries to run it with some super old version of gzip that isn't compatible...